### ISSUE TITLE
Change GPL to BSD?

### ISSUE THREAD

**Issue by skaurus** [Cerdic/jQl#2]
AFAIK I can't use this work or parts of it in commercial website without publishing all its source :-)

But in fact I want to, leaving credits for you and link to this repo, of course.

**Comment by cometta** [comment]
please provide more flexible licensing to encourage more people to use it

**Comment by lewisgoddard** [comment]
> ### GPL - Section 4. Conveying Verbatim Copies.
> 
> You may convey verbatim copies of the Program's source code as you receive it, in any medium, provided that you conspicuously and appropriately publish on each copy an appropriate copyright notice; keep intact all notices stating that this License and any non-permissive terms added in accord with section 7 apply to the code; keep intact all notices of the absence of any warranty; and give all recipients a copy of this License along with the Program.
> 
> You may charge any price or no price for each copy that you convey, and you may offer support or warranty protection for a fee.

Based on this, you can use to work inside your web-app without open-sourcing all your assets. The GPL only really instigates any kind of restrictions after modification.

_I am not a lawyer._

**Comment by wladston** [comment]
Also not a lawyer, but I believe that @lewisgoddard is wrong, and that the whole point of GPL is making anything that uses GPL code also open source.

This question talks about it: http://programmers.stackexchange.com/questions/47032/can-i-use-gpl-software-in-a-commercial-application

But a web page is always distributed with its source code for now, since javascript isn't served in compiled form. So I believe that using this on a website is legally OK. I'm not sure, though.

Using BSD style licenses would make it much easier for anyone that wants to use this code, though.

**Comment by lewisgoddard** [comment]
I agree that BSD or MIT licenses would be superior choices, but no-one has ever been successfully sued for running GPL JavaScript in a commercial browser, nor would JavaScript have an inference to server-side language opening.
